js/views/userPageVw.js

```diff
@@ -595,6 +607,11 @@ module.exports = Backbone.View.extend({
 
   storeClick: function(e){
     "use strict";
+    var select = document.getElementById('categories');
```

Comment:
Same note here about referencing the element consistently, use "js-categories" or "#categories" everywhere.